sites/all/modules/reol_app_feeds/src/Feed/FrontPageFeed.php

```diff
@@ -19,9 +19,14 @@ class FrontPageFeed extends AbstractFeed {
    *   The front page ids.
    */
   public static function getFrontPageIds() {
-    $value = _reol_app_feeds_variable_get('reol_app_feeds_frontpage', 'frontpage_ids', []);
-
-    return array_filter(array_values($value));
+    $group_name = 'reol_app_feeds_frontpage';
+    $field_name = 'page_ids';
+    $pages = _reol_app_feeds_variable_get($group_name, $field_name, []);
+    $included = array_filter($pages, function ($page) {
+      return isset($page['included']) && 1 === $page['included'];
```

Comment:
Magic no. 1 in expression?

Reply:
Well, it's the same `1` used throughout the code to check if a Drupal form checkbox has been checked.